package.json

```diff
@@ -46,7 +46,7 @@
 	"activationEvents": [
 		"*"
 	],
-	"main": "./dist/extension.js",
+	"main": "./out/extension.js",
```

Comment:
This was needed without it the metadata was missing. Probably need to update some additional tsconfig.json file/s